operators/foxglove/README.md

```
@@ -0,0 +1,466 @@
+| ---- | ---- | ----------- |
+| `input` | `holoscan::TensorMap` | Pose tensor containing either a row-major 4x4 transform matrix or seven `xyz+quat` values. |
+
+Outputs:
+
+| Port | Type | Description |
+| ---- | ---- | ----------- |
+| `messages` | `std::shared_ptr<FoxgloveBatch>` | `FrameTransform` batch suitable for `FoxglovePublisherOp`. |
+
+## Parameters
+
+### `FoxglovePublisherOp`
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)